frontend/src/Editor/Components/Button.jsx

```diff
@@ -28,18 +42,34 @@ export const Button = function Button({ height, properties, styles, fireEvent, r
     setLabel(text);
   });
 
+  registerAction('disable', async function (value) {
+    setDisable(value);
+  });
+
+  registerAction('visibility', async function (value) {
+    setVisibility(value);
+  });
+
+  registerAction('loading', async function (value) {
+    setLoading(value);
+  });
+
   return (
     <div className="widget-button">
       <button
-        disabled={disabledState}
+        disabled={disable}
         className={cx('jet-button btn btn-primary p-1 overflow-hidden', {
-          'btn-loading': loadingState,
+          'btn-loading': loading,
         })}
         style={computedStyles}
         onClick={(event) => {
           event.stopPropagation();
           fireEvent('onClick');
         }}
+        onMouseMove={(event) => {
```

Comment:
Using `onMouseMove` will trigger on every mouse movement, right? I think `onHover` then is not the right event to trigger. Instead it would be `onMouseEnter` or `onMouseOver` may be?